todoApp/spec/main.tsp

```diff
@@ -289,7 +289,7 @@ namespace TodoItems {
     op createFileAttachment(
       @header contentType: "multipart/form-data",
       @path itemId: TodoItem.id,
-      @body contents: TodoFileAttachment,
+      @multipartBody body: {contents: HttpPart<TodoFileAttachment>},
```

Comment:
follow https://github.com/allenjzhang/typespec-e2e-demo/pull/32/files#r1887960263

Reply:
nit, we may call it e.g. `FileAttachmentMultipartRequest`, to avoid using TCGC generated name.

Reply:
Updated